source/examples/functions/McePosTable/definition.yaml

```diff
@@ -1,6 +1,17 @@
 author: Rioual
 comment: Performed a trajectory stored in a PosTable
 changelog:
+  - version: 0.6.0
+    date: 2024-09-30
+    author: Rioual
+    fixed:
+      - State machine stuck if `nMode = 2` and `WAIT_FOR_STEP` actionID is active
+      - Motion and actionID are run again if servos are disabled then enabled
```

Comment:
Perhaps change this to "Motion and actionID no longer being re-executed after recovery of `bSystemReady`"

Reply:
Done in [b465bf2](https://github.com/YaskawaEurope/mlx-examples/pull/46/commits/b465bf2885f74496eda076449596517d9658da62)